client/src/components/Landing.js

```diff
@@ -2,52 +2,53 @@ import React, {Component} from 'react'
 import Card from './Card'
 import {NavLink} from 'react-router-dom'
 import './Landing.css'
+import {Row, Col, Container} from 'reactstrap'
 
 class Landing extends Component {
   render = () => [
-    <main key="main" role="main" className="container landing">
+    <Container tag="main" key="main" role="main" className="landing-page">
       <h1>
         <b>verejne.</b>digital
       </h1>
       <p className="lead">Aplikácia umelej inteligencie na dáta slovenských verejných inštitúcii</p>
-      <div className="row">
-        <div className="col">
+      <Row>
+        <Col className="col-lg-3 col-md-4 col-sm-6 card-wrapper">
```

Comment:
I'm not too big on bootstrap columns - and I supect the problems it currently has may be related

maybe a nicer behaviour would come from keeping bootstrap out of layout related business (use it just for 'styled components'), and do the layouting with just flexbox (and maybe some media queries)